packages/flutter/test/material/popup_menu_test.dart

```diff
@@ -851,6 +851,129 @@ void main() {
     expect(tester.getTopLeft(find.text('Item 0')).dx, 72);
     expect(tester.getTopLeft(find.text('Item 1')).dx, 72);
   });
+
+  test("PopupMenuButton's child and icon properties cannot be simultaneously defined", () {
```

Comment:
```suggestion
  test('PopupMenuButton's child and icon properties cannot be simultaneously defined', () {
```

Reply:
Rather have an escape charcter?

Reply:
The guide you linked states "Use double quotes for [...] strings that contain single quotes.". That was the reason I adjusted it. Let me know what you think, but the suggestion does not work because there is a single quote in the string.

Reply:
I did not notice the apostrophe/single quote in the string. This is fine :) Thank you!